### PR TITLE
Bookshelf features

### PR DESCRIPTION
**Pull request by cosmoscha**
I added delete buttons and fixed the unlimited ratings, please review

### CODE REVIEW

File: routes/bookshelf.js [hunk 1]
```diff
@@ -80,12 +79,23 @@ router.get(
 })
 )
 
-router.delete(
-  "/:status",
+router.post(
+  "/:bookId/delete",
+  requireAuth,
   asyncHandler(async (req, res) => {
+    console.log("hello")
```
Comment (WJVincent) [inline routes/bookshelf.js:80]: We will want to remove this console log

File: routes/bookshelf.js [hunk 2]
```diff
@@ -80,12 +79,23 @@ router.get(
...
+    const userId = await parseInt(req.session.auth.userId);
+    const bookId = parseInt(req.params.bookId, 10);
+    const userBookInstance = await db.UserBook.findOne({
+      where: { userId, bookId },
+    });
+      await userBookInstance.destroy();
+      // await db.UserBook.save();
```
Comment (WJVincent) [inline routes/bookshelf.js]: We will want to remove these comments

**Review by WJVincent — CHANGES_REQUESTED** [review]
Looks good, just some cleanup

File: routes/bookshelf.js (hunk 1 above)
Comment (cosmoscha) [reply routes/bookshelf.js:80]: done!

File: routes/bookshelf.js (hunk 2 above)
Comment (cosmoscha) [reply routes/bookshelf.js]: all cleaned up

**Review by WJVincent — APPROVED** [review]
Looks good

**Pull request merged**